stack/core/src/main/java/org/apache/usergrid/corepersistence/CpEntityManager.java

```diff
@@ -745,7 +745,32 @@ public RelationManager getRelationManager( EntityRef entityRef ) {
     @Override
     public Set<String> getApplicationCollections() throws Exception {
 
-        return getRelationManager( getApplication() ).getCollections();
+        Set<String> existingCollections = getRelationManager( getApplication() ).getCollections();
+
+        //Handles reading custom collections.
+//        Set<String> dynamic_collections = cast( getDictionaryAsSet( getApplicationRef(), Schema.DICTIONARY_COLLECTIONS ) );
```

Comment:
This was left here as a way to see how custom collections used to be queried but probably should be deleted and replaced with a jira ticket.